src/App.css

```diff
@@ -0,0 +1,46 @@
+/* .App {
+  text-align: center;
+} */
```

Comment:
Similar to debugging statements, we want to delete commented out code before PRs. If we are making frequent commits, we can always revisit the code in our git history.